app/src/features/rules/screens/templatesList/modals/TemplatePreviewModal/TemplatePreviewModal.tsx

```diff
@@ -1,23 +1,31 @@
 import React from "react";
 import { useDispatch, useSelector } from "react-redux";
 import { useNavigate } from "react-router-dom";
```

Comment:
Are we not using same modal in SharedList Rule viewer?

Reply:
No SharedList rule viewer modal is different